src/__tests__/add-relay-test-fixture-typings.test.ts

```diff
@@ -0,0 +1,11 @@
+import { defineTest } from "jscodeshift/src/testUtils"
+
+// const consoleLog = console.log
+// beforeAll(() => {
+//   console.log = jest.fn()
+// })
+// afterAll(() => {
+//   console.log = consoleLog
+// })
```

Comment:
Is this needed?

Reply:
It’s an easy way to silence all logging and focus on test failure output.